graph-builder/src/registry.rs

```diff
@@ -108,6 +109,8 @@ pub fn fetch_releases(
         .build()
         .map_err(|e| format_err!("{}", e))?;
 
+    let quay_client: quay::v1::Client = quay::v1::Client::builder().build()?;
```

Comment:
Can you please wire an `Option<String>` from main to the `.access_token()` on the builder here?

Reply:
I would not have this token provided as a CLI flag for security reasons, so I suggest to read it either from a file or an environment variable, whereas a file is preferable to me. I haven't tried the authentication against the quay API yet. What format do we expect this token to have?

Reply:
I do agree with all of the above (that is, storing it in a file). To be honest if we don't need it right I'm fine with having just a `None` at the top caller, but having the rest of the logic already wired-in for the time when that requirement arrives. The token itself is a plain 41-chars ASCII string, without any apparent structure.

Reply:
I've added all the infrastructure for it, although none of the tests work as we're lacking an API. The only concern I'd address in this PR if we expect the read of the file to work as is, e.g. do we have to strip EOL or EOF?

Reply:
I would suggest to use a line reader, pick the first line, and `trim_end` on it. That would help users to avoid invisible typos, and us to avoid sending weird characters over HTTP.